json-schemas/protocol-definition.json

```diff
@@ -4,29 +4,26 @@
   "type": "object",
   "additionalProperties": false,
   "required": [
-    "recordDefinitions",
+    "types",
     "records"
```

Comment:
change to `structure` or is that the next PR?

Reply:
That's in the next one